src/MathProg/formulation.jl

```diff
@@ -3,7 +3,8 @@ mutable struct Formulation{Duty <: AbstractFormDuty}  <: AbstractFormulation
     var_counter::Int
     constr_counter::Int
     parent_formulation::Union{AbstractFormulation, Nothing} # master for sp, reformulation for master
-    optimizer::AbstractOptimizer
+    moioptimizer::AbstractOptimizer
+    useroptimizer::AbstractOptimizer 
```

Comment:
```suggestion
    useroptimizer::Union{Nothing, UserOptimizer}
```
to make sure you receive a UserOptimizer

Reply:
ok

Reply:
Same here